servicetalk-http-api/src/main/java/io/servicetalk/http/api/HttpServerBuilder.java

```diff
@@ -322,6 +322,16 @@ public final HttpServerBuilder appendServiceFilter(final Predicate<StreamingHttp
      */
     public abstract HttpServerBuilder ioExecutor(IoExecutor ioExecutor);
 
+    /**
+     * Sets the {@link Executor} to use.
+     *
+     * @param executor {@link Executor} to use.
+     * @return {@code this}.
+     */
+    public HttpServerBuilder executor(Executor executor) {
+        throw new UnsupportedOperationException("Setting Executor not yet supported by " + getClass().getSimpleName());
```

Comment:
After looking at #1695, I've realized that we also need similar methods for gRPC builder. Can you please add those for 0.41 @bondolo?

Reply:
https://github.com/apple/servicetalk/pull/1804